examples/federated-learning/tff/nvflare/client1-deployment.yml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
```

Comment:
Can you add comments explaining why we skip these checks? Some of these seem valid.

Reply:
Done